### ISSUE TITLE
Cannot read property 'trim' of null

### ISSUE THREAD

**Issue by bernatfortet** [moroshko/react-autosuggest#278]
I'm getting a `Cannot read property 'trim' of null` error when I blur my input field.

1. The input is initially empty.
2. I focus the input
3. I blur the focus (without any change, that is, it's still empty)
4. then I get the error

What can I do about it?

**Comment by moroshko** [comment]
@bernatfortet I tried the steps you provided on [this Codepen](http://codepen.io/moroshko/pen/LGNJMy), but the error didn't appear.

Could you provide a Codepen that demonstrates the error?

**Comment by bernatfortet** [comment]
I can't recreate it in the Codepen so I guess it was on my side. Anyway, I've solved it.
I have my autosuggest as
`focusFirstSuggestion={true}`
And what happened is that I pressed enter after the first character type in the input. And the input was filled but the inputProp onBlur didn't return any "focusedSuggestion". So now I have a check that if "focusedSuggestion" is null but this.state.value has a value I should use this one.

```js
onBlur = (event, {focusedSuggestion}) => {
// Here
})
```

Anyway, thanks!
